Assignment-1-CPSC-121.cpp

```diff
@@ -0,0 +1,95 @@
+/* ----------------------------------------------------------------------------
+ * Copyright &copy; 2015 Samir Mandal (mandalsamir@fullerton.edu)
+ * Released under the [MIT License] (http://opensource.org/licenses/MIT)
+ * ------------------------------------------------------------------------- */
+
+/**
+ * This program will convert an integer input to Roman numeral
+ */
+
+#include <iostream>
+#include <string>
+
+using std::cout;
+using std::cin;
+using std::string;
+using std::endl;
+
+//Function declaration for integer to Roman number conversion
+string integerToRNumber (int number);
+
+// ----------------------------------------------------------------------------
+
+int main() {
+	int input_int;
+	string output_roman;
+	string user_option;
+
+	/*Block which makes function call to convert integer to Roman numeral and
+	exception handling for negative integers*/
+	block1:{
+		cout << "Enter a positive integer number ::";
+		cin >> input_int;
+		if (input_int < 0) {
+			cout << "Invalid input. Please enter \"Y\" in next prompt.\n";
+		}
+		else {
+		output_roman = integerToRNumber(input_int);
+		cout << "Roman numeral is ::" << output_roman << endl;
+			}
+		}
+
+			cout << "Do you like to convert another integer :: enter y/n::";
+			cin >> user_option;
+
+	// If the user enters 'n' or 'N': end the program ; otherwise goto block1
+			if (user_option == "n" || user_option == "N"){
+				exit(0);
+			}
+			else{
+				goto block1;
```

Comment:
lol -- i think we talked about how goto's probably aren't what you want to be using her.  not that they're bad in general, when they're necessary -- just that the code would be just as clean without them (and it would be easier for readers, and you, to verify that it was clean, if they weren't there).